src/dotnet/utils/JacobianComparisonLib/JacobianComparison.cs

```diff
@@ -63,15 +63,15 @@ public void CompareGmmFullAndPartGradients(string path1, string[] paths2)
             if (!File.Exists(path1))
             {
                 this.ParseError = true;
-                this.Error = $"File {path1} doesn't exist.";
+                this.Error = $"File {path1.Replace("\\", "/")} doesn't exist.";
```

Comment:
Maybe that's what you mean by "JSON generating by JacobianComparision util is repaired (now it is set paths in Error field value without backslashes)."

It should go in its own commit at least, probably its own PR.

Reply:
I've added these changes to the this PR because wrong generated JSON by JacobianComparision fluences on violation checking in graph plotting. When I say "wrong JSON generating" I mean creating JSON with string values with unrecognizable escape sequences (which are not an escape sequences exactly), e.g. "C:\Users\ADBench" (here "\U" and "\A" are understood as escape sequences, it makes JSON unreadable by serialization tools).

But now I think that you are right and such changes have to be at least in the separated commit. Thanks for noticing

Reply:
By the way, thanks for your great work on my PR! Now I see how bad it was to create one huge commit and to leave you dealing with it. I'm sorry for this and I hope I didn't waste your time a lot

Reply:
> I think that you are right and such changes have to be at least in the separated commit. Thanks for noticing

You are welcome.  I will submit a PR for this today (or you can cherry-pick the commit from my branch and submit a PR yourself if you want).

> By the way, thanks for your great work on my PR! Now I see how bad it was to create one huge commit and to leave you dealing with it. I'm sorry for this and I hope I didn't waste your time a lot

If it has helped your understanding then it was worth me doing the work!  Different reviewers like to see different things in PRs.  For me personally I prefer to have PRs (or at least commits) that contain only refactorings and trivial changes and others that contain only functionality change.  That makes it a lot easier to review.  In the case of this PR it's particularly important because the logic of the graph plotting code is quite convoluted.